frontends/web/src/routes/new-settings/components/dropdowns/activecurrenciesdropdown.module.css

```diff
@@ -0,0 +1,58 @@
+.select :global(.react-select__input-container) {
+    position: absolute;
+    top: -2px;
+    left: 6px;
+}
+
+.select :global(.react-select__option) {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+}
+
+/*displays only first 3 currencies, the rest gets hidden*/
+.select :global(.react-select__multi-value):nth-child(n + 4) {
+    display: none;
+}
+
+
+/*
+displays ', ' as a pseudo component for all currency component
+except the last displayed component (the 3rd currency component).
+
+The reason for :nth-last-child(2) instead of just :last-child is because there's an extra
+component created by react-select after the actual last currency component. 
+
+So, we're targetting the second to last component here, 
+which is the last selected currency in the DOM.
+*/
+.select:not(.hideMultiSelect) :global(.react-select__multi-value):not(:nth-last-child(2))::after {
+    content: ', ';
```

Comment:
looks like space doens't work it seems to collapse and be ignored in pseudo element via content.

It would look nicer with a bit of space after the comma, can you add `padding-right: 2px;`?

Reply:
Ooh lol good catch, I didnt even notice.. 🤔